### PR TITLE
Bump elementary LLC copyright year to 2018

### PR DESCRIPTION
**Pull request by jeremypw**

### CODE REVIEW

**Comment by danrabbit** [comment]
If we're touching all these lines I think we should update the link from launchpad to just our website as well

**Comment by jeremypw** [comment]
Done.  I left three links to launchpad.net/marlin

File: src/View/Window.vala [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
 * Copyright (c) 2010 Mathijs Henquet <mathijs.henquet@gmail.com>
-*               2017 elementary LLC. (https://elementary.io)
+*               2018 elementary LLC. (https://elementary.io)
```
Comment (danrabbit) [inline src/View/Window.vala]: This should probably be 2017-2018

File: src/View/Widgets/Label.vala [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (c) 2016 elementary LLC. (http://launchpad.net/pantheon-files)
+* Copyright (c) 2016 elementary LLC. <https://elementary.io>
```
Comment (danrabbit) [inline src/View/Widgets/Label.vala]: 2016-2018 :)

File: src/View/Slot.vala [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /***
-    Copyright (c) 2015-16 elementary LLC (http://launchpad.net/elementary)
+    Copyright (c) 2015-16 elementary LLC <https://elementary.io>
```
Comment (danrabbit) [inline src/View/Slot.vala]: 2015-2018

File: src/View/Miller.vala [hunk 4]
```diff
@@ -1,5 +1,5 @@
 /***
-    Copyright (c) 2015-16 elementary LLC (http://launchpad.net/elementary)
+    Copyright (c) 2015-16 elementary LLC <https://elementary.io>
```
Comment (danrabbit) [inline src/View/Miller.vala]: 2015-2018

File: src/Utils/Permissions.vala [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (c) 2016 elementary LLC. (http://launchpad.net/pantheon-files)
+* Copyright (c) 2016 elementary LLC. <https://elementary.io>
```
Comment (danrabbit) [inline src/Utils/Permissions.vala]: 2016-2018

File: src/IconRenderer.vala [hunk 6]
```diff
@@ -1,5 +1,5 @@
 /***
-    Copyright (c) 2016 elementary LLC (http://launchpad.net/elementary)
+    Copyright (c) 2016 elementary LLC <https://elementary.io>
```
Comment (danrabbit) [inline src/IconRenderer.vala]: 2016-2018

File: src/ClipboardManager.vala [hunk 7]
```diff
@@ -1,5 +1,5 @@
 /***
-    Copyright (c) 2016 elementary LLC (http://launchpad.net/elementary)
+    Copyright (c) 2016 elementary LLC <https://elementary.io>
```
Comment (danrabbit) [inline src/ClipboardManager.vala]: 2016-2018

File: libcore/tests/UtilTests/UtilTests.vala [hunk 8]
```diff
@@ -1,5 +1,5 @@
 /*
-* Copyright (c) 2017 elementary LLC
+* Copyright (c) 2018 elementary LLC
```
Comment (danrabbit) [inline libcore/tests/UtilTests/UtilTests.vala]: 2017-2018

File: libcore/tests/MarlinIconInfoTests/MarlinIconInfoTests.vala [hunk 9]
```diff
@@ -1,5 +1,5 @@
 /***
-    Copyright (c) 2017 elementary LLC
+    Copyright (c) 2018 elementary LLC
```
Comment (danrabbit) [inline libcore/tests/MarlinIconInfoTests/MarlinIconInfoTests.vala]: 2017-2018

File: libcore/tests/GOFFileTests/GOFFileTests.vala [hunk 10]
```diff
@@ -1,5 +1,5 @@
 /***
-    Copyright (c) 2017 elementary LLC
+    Copyright (c) 2018 elementary LLC
```
Comment (danrabbit) [inline libcore/tests/GOFFileTests/GOFFileTests.vala]: 2017-2018. Missing URL

File: libcore/tests/GOFDirectoryAsyncTests/GOFDirectoryAsyncTests.vala [hunk 11]
```diff
@@ -1,5 +1,5 @@
 /***
-    Copyright (c) 2017 elementary LLC
+    Copyright (c) 2018 elementary LLC
```
Comment (danrabbit) [inline libcore/tests/GOFDirectoryAsyncTests/GOFDirectoryAsyncTests.vala]: 2017-2018. Missing URL

File: libcore/Thumbnailer.vala [hunk 12]
```diff
@@ -1,5 +1,5 @@
 /***
-    Copyright (c) 2016 elementary LLC (http://launchpad.net/elementary)
+    Copyright (c) 2016 elementary LLC <https://elementary.io>
```
Comment (danrabbit) [inline libcore/Thumbnailer.vala]: 2016-2018

File: filechooser-module/FileChooserDialog.vala [hunk 13]
```diff
@@ -1,5 +1,5 @@
 /*-
- * Copyright (c) 2015-2016 elementary LLC (http://launchpad.net/elementary)
+ * Copyright (c) 2015-2016 elementary LLC <https://elementary.io>
```
Comment (danrabbit) [inline filechooser-module/FileChooserDialog.vala]: 2015-2018

**Comment by jeremypw** [comment]
Oops - I guess I assumed that everything had been updated to 2017 when searching  ; (.   Not sure why the Window.vala one was not found though.

**Pull request merged**